Updating camera page to better explain camera change methods

langsmith opened the pull request:
Per @zugaldia request, this pr resolves https://github.com/mapbox/android-docs/issues/1138 (made by @tsuz ) by clarifying the information about the camera update methods. This pr also cleans up the https://docs.mapbox.com/android/maps/overview/camera/ with other updates, helpful links, etc.

![screencapture-localhost-8080-android-maps-overview-camera-2019-11-18-14_52_41](https://user-images.githubusercontent.com/4394910/69101074-83cb0c00-0a13-11ea-9e5d-456088f9bd6f.png)

colleenmcginnis commented on src/pages/maps/overview/camera.md:
```diff
@@ -10,42 +10,15 @@ language:
 ---
 
-The Maps SDK's maps are represented as a flat plane using a [Mercator projection](https://en.wikipedia.org/wiki/Mercator_projection). On the east-west-axis, the map seamlessly wraps around on itself an infinite amount of times. Because the bottom and the top of the map could infinitely grow larger and further distort the map towards the poles, the north-south-axis gets cut off around the 90 degre ...
+The Maps SDK's maps are represented as a flat plane using a [Mercator projection](https://en.wikipedia.org/wiki/Mercator_projection). The map seamlessly wraps around on itself an infinite amount of times o the east-west-axis. Because the bottom and the top of the map could infinitely grow larger and further distort the map towards the poles, the north-south-axis gets cut off around 90 degrees nor ...
```
@danswick can you take a look at this paragraph? I'm having trouble refining the language we use to describe repeated world copies etc. I know you have more experience explaining projection (and projection-adjacent) concepts.

langsmith replied:
fyi, says `Outdated` above only because I fixed a missing letter typo.

danswick replied:
Thanks for the reminder, @langsmith! I'm not sure I understand why it's important to describe mercator, world copies, latitude limits, etc here. My understanding is that the camera concept has more to do with GL norms of representing user perspective in 3D space than with maps/projections. Unless folks feel like this projection-related info would help facilitate user understanding of the `camera` concept, I'd be in favor of removing it completely.

langsmith replied:
thanks @danswick . I've pushed a commit that removes the projection paragraph all together.

adrianababakanian commented on src/pages/maps/overview/camera.md:
```diff
@@ -10,42 +10,15 @@ language:
 ---
 
-The Maps SDK's maps are represented as a flat plane using a [Mercator projection](https://en.wikipedia.org/wiki/Mercator_projection). On the east-west-axis, the map seamlessly wraps around on itself an infinite amount of times. Because the bottom and the top of the map could infinitely grow larger and further distort the map towards the poles, the north-south-axis gets cut off around the 90 degre ...
+The Maps SDK's maps are represented as a flat plane using a [Mercator projection](https://en.wikipedia.org/wiki/Mercator_projection). The map seamlessly wraps around on itself an infinite amount of times on the east-west axis. Because the bottom and the top of the map could infinitely grow larger and further distort the map towards the poles, the north-south-axis gets cut off around 90 degrees no ...
```
> east-west axis 

> north-south-axis 

Probably would be a good idea to standardize the use of hyphens here.

langsmith replied:
Fixed

adrianababakanian commented on src/pages/maps/overview/camera.md:
```diff
@@ -10,42 +10,15 @@ language:
 ---
 
-The Maps SDK's maps are represented as a flat plane using a [Mercator projection](https://en.wikipedia.org/wiki/Mercator_projection). On the east-west-axis, the map seamlessly wraps around on itself an infinite amount of times. Because the bottom and the top of the map could infinitely grow larger and further distort the map towards the poles, the north-south-axis gets cut off around the 90 degre ...
+The Maps SDK's maps are represented as a flat plane using a [Mercator projection](https://en.wikipedia.org/wiki/Mercator_projection). The map seamlessly wraps around on itself an infinite amount of times on the east-west axis. Because the bottom and the top of the map could infinitely grow larger and further distort the map towards the poles, the north-south-axis gets cut off around 90 degrees no ...
```
This is a bit of a nit-pick, but I'd recommend changing "object's" to "object is." The usage of the apostrophe here confused me on my first read!

langsmith replied:
Fixed

adrianababakanian commented on src/pages/maps/overview/camera.md:
```diff
@@ -10,42 +10,15 @@ language:
+The camera can be tilted, rotated, zoomed, and/or moved by on-screen gestures such as pinching or with pre-written code. It's important to know that the camera object will not make any changes to markers, layers, sources, or other annotations that you've added without you directly influencing this behavior.
 
-A few camera event listeners are provided in the Maps SDK to notify when, for example, the camera's adjusted. You can read more about these listeners inside the [Events documentation](https://docs.mapbox.com/android/maps/overview/events/#camera-change-events).
+A few camera event listeners are provided in the Maps SDK to notify when, for example, the camera's adjusted. You can read more about these listeners inside the [events page](https://docs.mapbox.com/android/maps/overview/events/#camera-change-events).
```
Re: apostrophe usage mentioned above. Maybe "camera's" ⟶ "camera is".

langsmith replied:
Fixed

adrianababakanian commented on src/pages/maps/overview/camera.md:
```diff
@@ -10,42 +10,15 @@ language:
-}}
-
-Read all about [mapView XML attributes](https://docs.mapbox.com/android/maps/overview/#mapview-xml-attributes) to learn about setting the camera's initial position. You should either set the initial camera position in XML or through `MapboxMapOptions` to prevent unnecessary downloading of map tiles using up your user's data.
+The Maps SDK includes a `CameraPosition` class which comprises of the camera's target, angle, zoom, tilt, and padding. These camera options shape the user's perspective of the map tile(s).
```
"comprises" ⟶ "is comprised"

langsmith replied:
Fixed

adrianababakanian commented on src/pages/maps/overview/camera.md:
```diff
@@ -101,18 +126,14 @@ mapboxMap.animateCamera(CameraUpdateFactory.newCameraPosition(position), millise
 }}
 
-Aside from consuming a Camera Update object, a cancelable callback can be added to know when the animation finishes or if the user cancels the camera move by performing a gesture on the map. The ease and animate APIs have an optional duration parameter (in milliseconds) that lets you control the camera's animation duration.
+Aside from consuming a `CameraUpdate object`, a cancelable callback can be added to know when the animation finishes or if the user cancels the camera move by performing a gesture on the map. The `easeCamera()` or `animateCamera()` methods have an optional duration parameter (in milliseconds), which lets you control how quickly the camera travels to the new map location.
```
Was the intention here "`CameraUpdate` object" rather than "`CameraUpdate object`" (in reference to the inline code formatting)?

langsmith replied:
Yes. Fixed.

langsmith commented:
Thanks the review, @adrianababakanian . Addressed all of your feedback. Not urgent, but this pr is still waiting on some eyes from @danswick https://github.com/mapbox/android-docs/pull/1153#discussion_r348230646.
